homeassistant/components/mobile_app/http_api.py

```diff
@@ -89,7 +90,7 @@ async def post(self, request: Request, data: dict) -> Response:
             # If otherwise empty string contains emoji
             # use descriptive name of the first emoji
             data[ATTR_DEVICE_NAME] = emoji.demojize(
```

Comment:
Is this why we install and load the emoji package?!

Reply:
Haha yeah, its the only reason :)